src/GraphQL.Tests/Utilities/SchemaBuilderTestBase.cs

```diff
@@ -69,6 +71,11 @@ public ExecutionResult CreateQueryResult(string result)
             }
             return new ExecutionResult { Data = expected };
         }
+
+        protected string ReadSchema(string fileName)
+        {
+            return File.ReadAllText(Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Files", fileName));
```

Comment:
Consider using `AppDomain.CurrentDomain.BaseDirectory` instead of `Assembly.GetExecutingAssembly().Location`.